master/reference/calicoctl/resources/workloadendpoint.md

```diff
@@ -14,14 +14,15 @@ in a specific namespace only applies to the WorkloadEndpoint in that namespace.
 Two resources are in the same namespace if the namespace value is set the same
 on both.
 
-For `calicoctl` [commands]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/) that specify a resource type on the CLI, the following
-aliases are supported (all case insensitive): `workloadendpoint`, `workloadendpoints`, `wep`, `weps`.
+For `calicoctl` [commands]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/) 
+that specify a resource type on the CLI, the following aliases are supported (all case 
+insensitive): `workloadendpoint`, `workloadendpoints`, `wep`, `weps`.
 
 > **Note**: While `calicoctl` allows the user to fully manage Workload Endpoint resources,
 > the lifecycle of these resources is generally handled by an orchestrator-specific
 > plugin such as the Calico CNI plugin, the Calico Docker network plugin,
```

Comment:
Docker network plugin is also not supported in v3.0

Reply:
That's true, but this is the file in master. Am confining the deletions to the v3.0 directory.

However, you raise a good point that the note in the v3.0 directory discusses the Docker network plugin, which is not supported, so I've deleted this reference.